CONTRIBUTING.md

```diff
@@ -0,0 +1,140 @@
+# DeePMD-kit Contributing Guide
+
+Welcome to [DeePMD-kit](https://github.com/deepmodeling/deepmd-kit)! We are excited about the prospect of you joining [DeepModeling Community](https://github.com/deepmodeling/community/).
+
+## What you can contribute
+
+You can either make a code contribution, help improve our document or offer help to other users. Your help is always appreciated. Come and have fun!
+
+### Code contribution
+You can start from any one of the following items to help improve deepmd-kit
+
+- Smash a bug
+- Implement a feature or add a patch, whatever you think deepmd-kit is missing
+- Browse the deepmd-kit [roadmap](doc/roadmap.md) and pick a bug you want to smash or a feature you want to implement
```

Comment:
We do not put the bugs and feature requests in `doc/roadmap.md`, we put them in [issues](https://github.com/deepmodeling/deepmd-kit/issues)

Reply:
ok